Rules/PlaceOpenBrace.cs

```diff
@@ -344,17 +345,19 @@ private int GetStartColumnNumberOfTokenLine(Token[] tokens, int refTokenPos)
 
         private List<CorrectionExtent> GetCorrectionsForBraceShouldBeOnSameLine(
             Token precedingExpression,
+            Token optionalCommentOfPrecedingExpression,
             Token lCurly,
             string fileName)
         {
             var corrections = new List<CorrectionExtent>();
+            var optionalComment = optionalCommentOfPrecedingExpression != null ? $" {optionalCommentOfPrecedingExpression}" : string.Empty;
             corrections.Add(
                 new CorrectionExtent(
                     precedingExpression.Extent.StartLineNumber,
                     lCurly.Extent.EndLineNumber,
                     precedingExpression.Extent.StartColumnNumber,
                     lCurly.Extent.EndColumnNumber,
-                    precedingExpression.Text + " " + lCurly.Text,
+                    $"{precedingExpression.Text} {lCurly.Text}{optionalComment}",
```

Comment:
without knowing too much about this project... should there be a space here between the `}{`: 
`{lCurly.Text}{optionalComment}` for when `optionalComment` isn't empty string and actually is a comment?

Reply:
That is the point of having the separate optionalComment variable, if it is not an empty string, then the variable itself contains the preceding space character. This way I avoid appending a space character when there is no comment.